monochart/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.34
+version: 1.2.0
```

Comment:
Unfortunately with this monochart we don't have proper versioning. It has to be bumped by the patch version each time.